lightwood/mixer/neural.py

```diff
@@ -341,10 +343,7 @@ def __call__(self, ds: EncodedDs,
                 else:
                     decoded_prediction = self.target_encoder.decode(Yh, **kwargs)
 
-                if not self.timeseries_settings.is_timeseries or self.timeseries_settings.nr_predictions == 1:
-                    decoded_predictions.extend(decoded_prediction)
-                else:
-                    decoded_predictions.append(decoded_prediction)
+                decoded_predictions.extend(decoded_prediction)
```

Comment:
This solves the "edge case" behavior noted in an earlier comment by simplifying `Neural.__call__()`